hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/update/UpdateDecoder.java

```diff
@@ -110,7 +111,7 @@ private static boolean isKnownImmutable(@Nullable final Token token) {
     public @Nullable TransactionBody decodeTokenUpdateV1(@NonNull final HtsCallAttempt attempt) {
         final var call = UpdateTranslator.TOKEN_UPDATE_INFO_FUNCTION_V1.decodeCall(
                 attempt.input().toArrayUnsafe());
-        return decodeTokenUpdate(call, attempt.addressIdConverter());
+        return decodeTokenUpdate(call, attempt.addressIdConverter(), false);
```

Comment:
David will likely ask for an enum instead of using a boolean :-).  Is it possible to use an overloaded version of `decodeTokenUpdate` instead of adding these versions passing a `false`?

Reply:
Did a slight rework of the decoding. Please check again and DM me if something seems off